components/brave_wallet_ui/nft/components/nft-content/nft-content.tsx

```diff
@@ -33,15 +33,16 @@ export const NftContent = (props: Props) => {
 
   return (
     <>
-      {imageUrl
-        ? <Image
-          src={url}
-        />
-        : <>
-          {isLoading &&
-            <LoadingOverlay isLoading={isLoading}>
-              <LoadIcon />
-            </LoadingOverlay>
+      {isLoading ? (
+        <LoadingOverlay isLoading={isLoading}>
+          <LoadIcon />
+        </LoadingOverlay>
+      ) : (
```

Comment:
Might be cleaner to write a plain `if` for ``isLoading` and return early

Reply:
Good call! Done!